cns/restserver/api.go

```diff
@@ -1072,14 +1073,32 @@ func (service *HTTPRestService) getNumberOfCPUCores(w http.ResponseWriter, r *ht
 	logger.Response(service.Name, numOfCPUCoresResp, resp.ReturnCode, err)
 }
 
-func getInterfaceIdFromCreateNetworkContainerURL(
-	createNetworkContainerURL string) string {
-	return strings.Split(strings.Split(createNetworkContainerURL, "interfaces/")[1], "/")[0]
-}
+func getAuthTokenAndInterfaceIdFromNcURL(networkContainerURL string) (string, string, error) {
```

Comment:
This is technically returning interface id and auth token in that order. Since the return types of the func are repeated strings, it's not clear from the call site what each actually represents. I'd suggest changing the function name to reflect the order of the returns and name the returns in the function signature, or have the function return a type that makes it very explicit, along the lines of:
```go
type ncURLParams struct {
    ncID        string
    interfaceID string
    authToken   string
}
```

Reply:
Changed result to a struct